### PR TITLE
Issue/486 editor explain limit on images

### PR DESCRIPTION
**Pull request by SergioEstevao**
Fix for https://github.com/wordpress-mobile/WordPress-iOS-Editor/issues/486

cc @bummytime, can i bother you with a review?

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m [hunk 1]
```diff
@@ -1071,7 +1071,18 @@ - (void)assetsPickerController:(CTAssetsPickerController *)picker didFinishPicki
 - (BOOL)assetsPickerController:(CTAssetsPickerController *)picker shouldSelectAsset:(ALAsset *)asset
 {
     if ([asset valueForProperty:ALAssetPropertyType] == ALAssetTypePhoto) {
-        return picker.selectedAssets.count < MaximumNumberOfPictures;
+        // If the image is from a shared photo stream it may not be available locally to be used
+        if (!asset.defaultRepresentation) {
+            [WPError showAlertWithTitle:NSLocalizedString(@"Cannot select this image", @"The title for an alert that says the image the user selected isn't available.")
+                                message:NSLocalizedString(@"This image is currently not available locally to be used in your post. Please try again later.", @"User information explaining that the image is not available locally. This is normally related to share photo stream images.")];
```
Comment (bummytime) [inline WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m]: @meremagee just a copy check here. Are you ok with this?

File: WordPress/Classes/ViewRelated/Post/WPPostViewController.m [hunk 2]
```diff
@@ -2017,9 +2017,14 @@ - (BOOL)assetsPickerController:(CTAssetsPickerController *)picker shouldSelectAs
                                 message:NSLocalizedString(@"This image is currently not available locally to be used in your post. Please try again later.", @"User information explaining that the image is not available locally. This is normally related to share photo stream images.")];
```
Comment (bummytime) [inline WordPress/Classes/ViewRelated/Post/WPPostViewController.m]: @meremagee another copy check (same as above). Please ignore me if this has already been hashed over! :grinning:

**Comment by bummytime** [comment]
Code looks good @SergioEstevao. I added some comments regarding copy that @meremagee can talk to if needed.

File: WordPress/Classes/ViewRelated/Post/WPPostViewController.m (hunk 2 above)
Comment (meremagee) [inline WordPress/Classes/ViewRelated/Post/WPPostViewController.m]: as suggested in [previous pr](https://github.com/wordpress-mobile/WordPress-iOS/pull/3028#discussion_r22801583), pls standardize to: 

@"This Photo Stream image cannot be added to your WordPress. Try saving to your Camera Roll before uploading."

> OK

File: WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m (hunk 1 above)
Comment (meremagee) [inline WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m]: as suggested in [previous pr](https://github.com/wordpress-mobile/WordPress-iOS/pull/3028#discussion_r22801583), pls standardize to: 

@"This Photo Stream image cannot be added to your WordPress. Try saving to your Camera Roll before uploading."

> OK

**Comment by bummytime** [comment]
@SergioEstevao, so after you make the two copy adjustments above as well as the stuff here:

https://github.com/wordpress-mobile/WordPress-iOS-Editor/issues/486#issuecomment-70291732

> Short-term if faster/easier: alert. Let's go with single-button alert as @bummytime suggests.
>
>message: You can only add 10 photos at a time
>confirm/dismiss: OK

I say :shipit:

**Pull request merged**